code/game/gamemodes/nuclear/nuclearbomb.dm

```diff
@@ -415,7 +415,16 @@ GLOBAL_VAR(bomb_set)
 	if(timing)	//boom
 		INVOKE_ASYNC(src, .proc/explode)
 		return
-	qdel(src)
+
+    //if no boom then we need to let the blob capture our nuke
+	var/turf/T = get_turf(src)
+	if(!T)
+		return
+	if(locate(/obj/structure/blob) in T)
+		return
+	var/obj/structure/blob/captured_nuke/N = new(loc, src)
```

Comment:
```suggestion
	var/obj/structure/blob/captured_nuke/N = new(T, src)
```
If by some miracle a blob manages to eat a nuke thats in a locker or some other non-turf loc.